### DISCUSSION TITLE
Proyek anime list pake api hasil scraping

### DISCUSSION THREAD

**Discussion by sandigloomy** [deaafrizal/forum#70] [My Projects]
liat tutorial bang dea bikin cuyanimelist (btw masi baru sampe sesi 3) gw ubah api nya pake hasil scrapping dari web anime lain dan berhasil walau banyak problem solvingnya, ini baru pertama kali yoba next js pertama kali, tapi gw pernah nyoba tapi ngedit text doang buat portfolio,  sekarang gw lagi belajar golang

![Screenshot (195)](https://github.com/deaaprizal/forum/assets/92760335/fdc36162-5f01-48a1-8197-3f01ff826032)

walau kodingannya kacau kaya gini dan berhasil wkwk seneng sih

![Screenshot (198)](https://github.com/deaaprizal/forum/assets/92760335/daf81666-0938-4758-92ca-a66048b78ba7)

**Comment by deaafrizal** [comment]
The best bro. lanjutin terus belajar dan buat karya karyanya 🍻
